backend/Controllers/PropertiesController.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BackendApi.Data;
+using Property = BackendApi.Models.Property;
+using PropertyDetail = BackendApi.Models.PropertyDetail;
+using Entity = BackendApi.Data.Entities;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace BackendApi.Controllers
+{
+  /// <summary>
+  /// Provides endpoints for managing properties
+  /// </summary>
+  [Authorize]
+  [ApiController]
+  [Route("/api/[controller]")]
+  public class PropertiesController : ControllerBase
+  {
+    #region Variables
+    private readonly ILogger<PropertiesController> _logger;
+    private readonly IConfiguration _configuration;
+    private readonly GeoSpatialContext _dbContext;
+    #endregion
+
+
+
+    #region Constructors
+    /// <summary>
+    /// Creates a new instance of a PropertiesController class.
+    /// </summary>
+    /// <param name="logger"></param>
+    /// <param name="configuration"></param>
+    /// <param name="dbContext"></param>
+    public PropertiesController(ILogger<PropertiesController> logger, IConfiguration configuration, GeoSpatialContext dbContext)
+    {
+      _logger = logger;
+      _configuration = configuration;
+      _dbContext = dbContext;
+    }
+    #endregion
+
+    #region Endpoints
+
+    /// <summary>
+    /// Get a list of properties that are within the provided bounds
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet]
+    public IActionResult GetProperties(double? neLat, double? neLong, double? swLat, double? swLong)
```

Comment:
I like the "rectangular" filter (4 corners) better than the "circular" filter (center + diameter) because the most intuitive way of selecting on a map is to click, drag, click to create a rectangle with your area of interest.

Having users click a center-point and then having them drag to choose how big the circle should be is not the best UX I've seen

Reply:
The idea behind this is based on the Google maps API and Leaflet API. Both use the concept of bounds when fetching map data. Primary use case is fetching points based on the current viewport, which I'm assuming we would be doing as well (rather then getting the user to scroll the map and then create an area to request pins for).

Reply:
Sounds good to me

Reply:
👍